### PR TITLE
Initial Project Push

### PR DESCRIPTION
**Pull request by sharadshah1234**
Feature Branch created for Diamond DB with 3 Functionalities.
1. Display all Diamonds
2. Add Diamonds by Supplier
3. Delete a Supplier

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shahmitesh51) [inline .classpath]: please remove this file

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+spring.data.mongodb.database=DiamondMarket
```
Comment (Vishalpopat) [inline src/main/resources/application.properties:4]: need to add context path

File: src/main/java/com/diamondmarket/diamonds/api/DiamondApiController.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.diamondmarket.diamonds.model.Error;
+import com.diamondmarket.diamonds.model.*;
+import com.diamondmarket.diamonds.service.DiamondService;
+
+@RestController
+@SuppressWarnings("rawtypes")
```
Comment (Vishalpopat) [inline src/main/java/com/diamondmarket/diamonds/api/DiamondApiController.java]: Check for @SuppressWarnings , remove of not needed 
successResponse  modify line: Response<Object> response = new Response<>();

**Review by Vishalpopat — COMMENTED** [review]
swagger implementation left

File: src/main/resources/application.properties (hunk 2 above)
Comment (sharadshah1234) [reply src/main/resources/application.properties:4]: Done

**Pull request merged**